index.rst

```diff
@@ -40,6 +40,11 @@ fundamental array data structure), see `scipy.org <http://www.scipy.org/>`__.
 
 .. raw:: html
 
-   <div style="padding-top: 40px;text-align:center;">
-     <script type="text/javascript" src="http://www.ohloh.net/p/4894/widgets/project_partner_badge.js"></script>
-   </div>
+<div style="padding-top:40px; text-align:center; margin:0px auto;">
```

Comment:
I think this might need to be indented to actually work, but I'm no expert at RST.